GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -266,6 +269,10 @@
         CODE_LABEL.put("en", "Geo Code");
         CODE_LABEL.put("es", "Código Geo");
 
+        ACC_LABEL = new HashMap<String, String>();
+        ACC_LABEL.put("en", "Accuracy (m)");
+        ACC_LABEL.put("es", "Precisi�n (m)");
```

Comment:
I am getting an error when compiling this line `error: unmappable character for encoding UTF8`